gulp/tasks/e2eTests.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function (gulp, config, plugins) {
+
+    gulp.task('connectDist', function (done) {
+        plugins.connect.server({
+            root: 'dist/',
+            port: 8888
+        });
+    });
+
+    gulp.task('testE2e', function (done) {
```

Comment:
Is it possible to combine both the tasks ? 
I mean just by running single command, both the tests get executed ?
Something like 'gulp test' would run both the unit and e2e tests